clustered/client/src/test/java/org/ehcache/clustered/client/HAValidXmlTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.clustered.client;
+
+import org.ehcache.clustered.client.config.ClusteringServiceConfiguration;
+import org.ehcache.core.internal.service.ServiceLocator;
+import org.ehcache.xml.XmlConfiguration;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertThat;
+
+/**
+ * HAValidXmlTest
+ */
+public class HAValidXmlTest {
+
+  @Test
+  public void testCanLoadXmlConfigValidForHA() {
```

Comment:
I answered my own question - since any pattern validation is reported through the same exception, a single test is enough and has been added.